src/Microsoft.DocAsCode.Dfm/DfmCodeRenderer.cs

```diff
@@ -82,7 +82,9 @@ public virtual StringBuffer RenderFencesCode(DfmFencesToken token, Options optio
 
         public virtual StringBuffer RenderOpenPreTag(StringBuffer result, DfmFencesToken token, Options options)
         {
-            return result + "<pre>";
+            result += "<pre";
+            result = HtmlRenderer.AppendSourceInfo(result, options, token);
```

Comment:
SB HtmlRenderer.AppendSourceInfo(SB, ...)

---
In reply to: [114698165](https://github.com/dotnet/docfx/pull/1624#discussion_r114698165) [](ancestors = 114698165)